src/transformers/generation_utils.py

```diff
@@ -653,6 +654,7 @@ def _get_logits_processor(
         repetition_penalty: float,
         no_repeat_ngram_size: int,
         encoder_no_repeat_ngram_size: int,
+        input_ids: torch.LongTensor,
```

Comment:
Sorry to only notice this now. I think we can make this clearer here. We don't actually need the `input_ids`, but all we need is the seq_length of the input_ids here. Also it would help the reader here to know that it corresponds to the sequence_length of the input to the decoder (in case it's an encoder-decoder model).

Could you maybe instead create a variable `input_ids_seq_length` and pass this one instead?